extension/lib/magento/customer/getItems.js

```diff
@@ -0,0 +1,27 @@
+const MagentoRequest = require('../Request')
+
+/**
+ * @param {Object} context
+ * @param {Object} input
+ * @returns {Promise<{productIds: string[]}>}
+ */
+module.exports = async (context, input) => {
+  const request = new MagentoRequest(context, input.token)
+  const wishlistIdsEndpointUrl = `${context.config.magentoUrl}/wishlists`
+  /**
+   * Get all wishlist ids of the customer. At the moment we only support one wishlist per customer
+   * @typedef {Object} wishlists
+   * @property {string} wishlist_id
+   */
+  const wishlists = await request.send(wishlistIdsEndpointUrl, 'getWishlistItems')
+
+  if (!wishlists.length || !wishlists[0].wishlist_id) {
+    return { productIds: [] }
+  }
+
+  const wishlistItemsEndpointUrl = `${context.config.magentoUrl}/wishlists/${wishlists[0].wishlist_id}/items`
+  const wishlistItems = await request.send(wishlistItemsEndpointUrl, 'Request to Magento: getFavorites', 'GET')
```

Comment:
You could also think about introducing a dedicated getWishlistItems(wishlistId) function (e. g.) to the request class and moving line22+23 + the whole endpoint consts over there and simply call it here.

Reply:
Just fyi, the suggestion is a TD to remove that kind of implementation in Shopify. Bad architectural practice to over-bloat the api class. 
However, I could refactor this into some kind of factory. This may take a day of work though, and since the sprint is ending soon I will do it in the next story.